course/src/main/java/org/openedx/course/presentation/dates/CourseDatesScreen.kt

```diff
@@ -330,18 +255,8 @@ internal fun CourseDatesScreen(
                 Box(
                     Modifier
                         .fillMaxWidth()
-                        .pullRefresh(pullRefreshState)
                 ) {
                     when (uiState) {
-                        is DatesUIState.Loading -> {
```

Comment:
Did we handle this?

Reply:
Yes, we send an event to the container when the data is loading. I did not remove the "Loading" state since we need some state while the data is loading. However, I can rename it to "Empty" or make flow nullable as a variant.

Reply:
Got it! Please revisit it because we are only utilizing that event for CourseOutline. 

> However, I can rename it to "Empty" or make flow nullable as a variant
And no need for this. Just leave it as it is.